integration_tests/helpers/envHelper.ts

```diff
@@ -1,7 +1,11 @@
-function resourcePrefix(): string {
-  return `di-btm${
+const resourcePrefix = (): string =>
+  `di-btm${
     process.env.ENV_NAME !== undefined ? "-" + process.env.ENV_NAME : ""
   }`;
-}
 
-export { resourcePrefix };
+const configStackName = (): string =>
+  `di-btm-config-${
+    process.env.CONFIG_NAME !== undefined ? process.env.CONFIG_NAME : "dev"
+  }`;
+
+export { resourcePrefix, configStackName };
```

Comment:
nit pick but why not just `export` each `const`?

Reply:
I'd prefer that as well, but that's how it's done across the codebase, so I didn't mess with it.

Reply:
![okily-dokily](https://user-images.githubusercontent.com/35689158/209329306-874c50c2-3c4b-4b4d-a410-4fc6f53931e9.gif)